specialized/async_specialized_form_parser.ipynb

```diff
@@ -0,0 +1,222 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "L0J138mj7p1s"
+   },
+   "source": [
+    "# Document AI Specialized Parser (Async)\n",
+    "This notebook shows you how use Document AI's specialized parsers ex. Invoice, Receipt, and W9 asynchronously."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "_v0XtSwn7fmN",
+    "outputId": "373b9379-f6ac-451a-e428-ad9219fb31d0"
+   },
+   "outputs": [],
+   "source": [
+    "# Install necessary Python libraries\n",
+    "!pip install google-cloud-documentai\n",
+    "!pip install 'google-cloud-storage==1.31.0'\n",
+    "\n",
+    "!apt-get update"
```

Comment:
Do we really need this like "!apt-get update"

Reply:
Instead we should either

1) Let's add a requirements.txt in the top directory with the couple lines below and add instructions to download the libraries in the README?

google-cloud-documentai==0.3.0
google-cloud-storage==1.36.2

2) Add a text above saying "Install the libraries and restart the kernel." Put this all under a "Setup" section. We can also add some validations like:

Validate storage version is >=1.36.2
`!python3 -c "from google.cloud import storage; print('Storage version: {}'.format(storage.__version__))"`

Reply:
First option sounds great, will add requirements.txt in the PR

Reply:
works for me! but maybe say here to make sure that they run python -m pip install -r  requirements.txt from the root directory.

Reply:
or add this as a cell !python -m pip install -r ../requirements.txt

Reply:
Did the API change between 1.31 & 1.36.2? I'm getting:
TypeError: __init__() got an unexpected keyword argument 'client_options'
on the storage client with 1.36